test/browser/time-picker/time-picker-spec.js

```diff
@@ -38,8 +41,11 @@ describe('TimePicker', () => {
     );
 
     const timeTextField = TestUtils.findRenderedComponentWithType(render, TextField);
+    const timepicker = TestUtils.findRenderedComponentWithType(render, TimePicker);
 
-    expect(timeTextField.props.value, formatTime(initialTime));
+    expect(timepicker.state.time).to.be.equal(initialTime);
```

Comment:
15 lines before we're using `formatTime` for the assertion, is there any reason why we can't keep it consistent here?